docs-markdown/src/controllers/master-redirect-controller.ts

```diff
@@ -40,12 +40,13 @@ export class RedirectionFile {
     // Members mapping to JSON elements in master redirection file
     public source_path: string;
     public redirect_url: string;
-    public redirect_document_id: boolean = false;
+    public redirect_document_id: boolean;
```

Comment:
Just a suggestion because the below works, but couldn't we initialize this as something? The null check on 126 could just check for something else. Anyway, going to approve, but leave this for rumination.